.github/workflows/docbuild-and-upload.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       run: cp doc/cheatsheet/Pandas_Cheat_Sheet* web/build/
 
     - name: Upload web
-      run: rsync -az --delete --exclude='pandas-docs' --exclude='docs' web/build/ web@${{ secrets.server_ip }}:/var/www/html
+      run: rsync -az --delete --exclude='pandas-docs' --exclude='docs' --exclude='benchmarks' web/build/ web@${{ secrets.server_ip }}:/var/www/html
```

Comment:
Is `benchmarks` going to be created or did you mean `asv_bench`?

Reply:
This is not on the pandas repo. It's in the rendered website directory. So, next to `index.html`, `about/team.html`... The benchmarks server will be creating and pushing a `benchmarks` directory with the html files that asv renders with the results.

@DeaMariaLeon and myself will be explaining and documenting all the new benchmark stuff we're building. We'll also be asking what improvements people would like to see in the benchmarks infrastructure once we replicate more or less what we had until now but with our OVH server and a bit simpler way to run things.